README.md

```diff
@@ -15,6 +15,15 @@ products and subscriptions, etc.
 
 ## Getting started
 
+You can do local development using our `dev` or `pre` environment.
+
+Since browsers started to blocking Third-Party Cookies, your top domain from local machine needs to
+match yours session service top domain. Otherwise session service cookie will be Third-Party Cookie,
+and will not be send with XHR request.
+
+For example if your `pre` domain is pre.sdk-example.com, and it uses id.pre.sdk-example.com session
+service domain, your local domain should be local.sdk-example.com.
```

Comment:
This needs to be `local.pre.sdk-example.com` (subdomain of `pre.sdk-example.com`).

Reply:
Might also be worth mentioning that you require local https to get this working (if I am not mistaken).

Reply:
We have verified that `local.sdk-example.com` is enough. Works without https for me